metadata-ingestion/src/datahub/ingestion/source/looker/lookml_config.py

```diff
@@ -47,6 +45,9 @@
 @dataclass
 class LookMLSourceReport(StaleEntityRemovalSourceReport):
     git_clone_latency: Optional[timedelta] = None
+    looker_query_api_latency: Optional[Dict[str, timedelta]] = (
```

Comment:
this may become a large dict, should it just be a `TopKDict`?

we may also track successful/error api request counts